numalogic/registry/mlflow_registry.py

```diff
@@ -17,6 +17,7 @@
 
 import mlflow.pyfunc
 import mlflow.pytorch
+import mlflow.sklearn
```

Comment:
As discussed, I don't think we need this anymore.
Just import mlflow should work.